relay-server/src/middlewares/metrics.rs

```diff
@@ -26,7 +29,7 @@ pub async fn metrics(mut request: Request, next: Next) -> Response {
     let response = next.run(request).await;
 
     relay_statsd::metric!(
-        timer(RelayTimers::RequestsDuration) = start_time.into_inner().elapsed(),
+        timer(RelayTimers::RequestsDuration) = request_start.elapsed(),
```

Comment:
Since we now use `received_at` as `DateTime` I wanted to still use an `Instant` for monitoring the request duration.